omnibus/package-scripts/publish_image.sh

```diff
@@ -5,6 +5,7 @@ set -xe
 IMAGE_TAG=$1
 IMAGE_REPO=$2
 ARTIFACT_PATH=$3
+PUSH_LATEST=$4
```

Comment:
can we default to `false` so we don't need to pass it for `pre_release_image`

what you think?

Reply:
updated false to default value for pre release